### PR TITLE
Admin API Login

### PR DESCRIPTION
**Pull request by eladbash**

### CODE REVIEW

File: Admin/Client/src/api/wtAdminApi.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import axios from 'axios';
+import axios, { AxiosError } from 'axios';
 
 const requestPath = '/request';
+const authenticationPath = '/authentication';
```
Comment (ChengaDev) [inline Admin/Client/src/api/wtAdminApi.tsx:4]: isnt it declared in the config file?

File: Admin/Client/src/authentication/index.tsx [hunk 2]
```diff
@@ -26,6 +26,7 @@ export default class Authentication {
     }
 
     private static SetToken(token: string) {
-        this._cookies.set(this.COOKIE_NAME, token);
+        const msMonth = 1000 * 60 * 60 * 24 * 30;
+        this._cookies.set(this.COOKIE_NAME, token, { path: '/', expires: new Date(Date.now() + (msMonth)) });
```
Comment (ChengaDev) [inline Admin/Client/src/authentication/index.tsx:30]: consider give more descriptive name

File: Admin/routers/AuthenticationRouter.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { Request, Response, Router } from 'express';
+import { appConfig } from '../../Config/globalConfig';
+
+export function authenticationRouter() {
+    let router: Router = Router();
+
+    // Create new request
+    router.post(`/${appConfig.adminserver.routes.authentication}`, (req: Request, res: Response) => {
+        const username = req.body.username;
```
Comment (ChengaDev) [inline Admin/routers/AuthenticationRouter.ts:9]: why not destructuring?

File: Admin/Client/src/api/wtAdminApi.tsx (hunk 1 above)
Comment (eladbash) [reply Admin/Client/src/api/wtAdminApi.tsx:4]: as we spoke - it does but you can't import it because it's CRA

**Pull request merged**